config.mk

```diff
@@ -51,8 +51,8 @@ ENABLE_X11= -DENABLE_X11
 endif
 
 # flags
-DEFAULT_CPPFLAGS = -Wno-gnu-zero-variadic-macro-arguments -D_DEFAULT_SOURCE -DVERSION=\"${VERSION}\" -DSYSCONFDIR=\"${SYSCONFDIR}\"
-DEFAULT_CFLAGS   = -g -std=gnu11 -pedantic -Wall -Wno-overlength-strings -Os ${ENABLE_WAYLAND} ${ENABLE_X11} ${EXTRA_CFLAGS}
+DEFAULT_CPPFLAGS = -Wno-gnu-zero-variadic-macro-arguments -D_DEFAULT_SOURCE -DVERSION=\"${VERSION}\" -D_CCDATE="$(shell date '+%h %d %Y')" -DSYSCONFDIR=\"${SYSCONFDIR}\" ${ENABLE_WAYLAND} ${ENABLE_X11}
```

Comment:
`date +'%h %d %Y'` is using the locale dependent `%h`. Maybe use ISO8601 here: `date +%Y-%m-%d`?